models/error.go

```diff
@@ -267,6 +267,21 @@ func (err ErrReachLimitOfRepo) Error() string {
 	return fmt.Sprintf("user has reached maximum limit of repositories [limit: %d]", err.Limit)
 }
 
+// ErrReachLimitOfPrivateRepo represents a "ReachLimitOfRepo" kind of error.
```

Comment:
I know that most of the documentation in this file is absolutely terrible but this comment rather opaque.

```suggestion
// ErrReachLimitOfPrivateRepo represents an error due to reaching the limit of allowed private repositories for this user.
```